test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests.csproj

```diff
@@ -1,5 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <!-- Suppress warnings about lowercase variable names in generated code -->
+    <NoWarn>$(NoWarn);CS8981</NoWarn>
```

Comment:
Let's move the investigation to a new issue. Somehow Alpine doesn't still like that fixed(?) Grpc version 🤔

Reply:
As I remember it is a new feature in .NET 7 SDK. Which disallows lower-cased names. Grpc generates ithat variables, so me have warnings. I do not found any way to solve it in OTel AutoInstrumentation.

Reply:
Just the Alpine doesn't like it, maybe it's the native part that's not updated (as we do install native grpc packages also in the Dockerfile).